src/autopas/pairwiseFunctors/FlopCounterFunctor.h

```diff
@@ -125,7 +125,108 @@ class FlopCounterFunctor : public Functor<Particle, ParticleCell> {
   void SoAFunctor(SoA<SoAArraysType> &soa,
                   const std::vector<std::vector<size_t, autopas::AlignedAllocator<size_t>>> &neighborList, size_t iFrom,
                   size_t iTo, bool newton3) override {
-    utils::ExceptionHandler::exception("Functor::SoAFunctor(verlet): not yet implemented");
+    auto numParts = soa.getNumParticles();
+
+    if (numParts == 0) return;
+
+    double *const __restrict__ xptr = soa.template begin<Particle::AttributeNames::posX>();
+    double *const __restrict__ yptr = soa.template begin<Particle::AttributeNames::posY>();
+    double *const __restrict__ zptr = soa.template begin<Particle::AttributeNames::posZ>();
+
+    for (unsigned int i = iFrom; i < iTo; ++i) {
+      const size_t listSizeI = neighborList[i].size();
+      const size_t *const __restrict__ currentList = neighborList[i].data();
+
+      // this is a magic number, that should correspond to at least
+      // vectorization width*N have testet multiple sizes:
+      // 4: does not give a speedup, slower than original AoSFunctor
+      // 8: small speedup compared to AoS
+      // 12: highest speedup compared to Aos
+      // 16: smaller speedup
+      // in theory this is a variable, we could auto-tune over...
+#ifdef __AVX512F__
+      // use a multiple of 8 for avx
+      const size_t vecsize = 16;
+#else
+      // for everything else 12 is faster
+      const size_t vecsize = 12;
```

Comment:
did you check these values? or did you just copy the values of the ljfunctor?

Reply:
that's just copy & paste. Will investigate on monday.

Reply:
Quick Testingof one iteration of the flopcounter after merging #78 into this branch:

OMP_NUM_THREADS=1
./md-flexible
  --container
  XXXXXX
  --data-layout
  XXX
  --functor
  lj
  --iterations
  0
  --particles-per-dimension
  50
  --particle-spacing
  .4
  --cutoff
  1.
  --traversal
  c08,slice
  --verlet-rebuild-frequency
  5
  --verlet-skin-radius
  0

container  |   datalayout   | milliseconds | vecsize
------------ | ------------- | ------------- | -------------
LinkedCell |  aos | 98 | n/a
LinkedCell |  soa | 42 | 12
VerletLists |  aos | 36 | n/a
VerletLists |  soa | 30 |      4
VerletLists |  soa | 30 |      8
VerletLists |  soa | 29 |     12
VerletLists |  soa | 31 |     16